### PR TITLE
Add CD 73.0.3683.68 and fix handling of versions for new naming system

### PR DESCRIPTION
**Pull request by imurchie**
The Chromedriver team decided to be obnoxious and change the version naming scheme, to be in line with Chromium. One assumes this makes sense for them.

For us, it complicates the handling of semver. Comparisons now go through the [compare-versions](https://www.npmjs.com/package/compare-versions) package.

[Notes](https://chromedriver.storage.googleapis.com/73.0.3683.68/notes.txt) for version `73.0.3683.68`:
```
----------ChromeDriver 73.0.3683.68 (2019-03-06)----------
Supports Chrome version 73
Resolved issue 2768: ChromeDriver produces unexpected debug.log file if verbose logging is enabled [[Pri-1]]
Resolved issue 2728: Is Element Displayed command does not work correctly with v0 shadow DOM inserts [[Pri-1]]
Resolved issue 2799: Recent change to WebDriver Tests causes ChromeDriver to fail W3C Tests [[Pri-2]]
Resolved issue 2744: Execute Script returns wrong error code when JavaScript returns a cyclic data structure [[Pri-2]]
Resolved issue 1529: OnResponse behavior can lead to port exhaustion [[Pri-2]]
Resolved issue 2736: Close Window command should handle user prompts based on session capabilities [[Pri-2]]
Resolved issue 1963: Sending keys to disabled element should throw Element Not interactable error [[Pri-2]]
Resolved issue 2679: Timeout value handling is not spec compliant [[Pri-2]]
Resolved issue 2002: Add Cookie is not spec compliant [[Pri-2]]
Resolved issue 2716: Clearing Text Boxes [[Pri-3]]
Resolved issue 2714: ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:15756. Could not start driver. [[Pri-3]]
Resolved issue 2722: Execute Script does not correctly convert document.all into JSON format [[Pri-3]]
Resolved issue 2681: ChromeDriver doesn't differentiate "no such element" and "stale element reference" [[Pri-3]]
```

### CODE REVIEW

File: lib/chromedriver.js [hunk 1]
```diff
@@ -84,10 +86,7 @@ function getMostRecentChromedriver (mapping = CHROMEDRIVER_CHROME_MAPPING) {
   if (_.isEmpty(mapping)) {
     throw new Error('Unable to get most recent Chromedriver from empty mapping');
   }
-  return _.keys(mapping)
-    .map(semver.coerce)
-    .sort(semver.rcompare)
-    .map((v) => `${semver.major(v)}.${semver.minor(v)}`)[0];
+  return _.last(_.keys(mapping).sort(compareVersions));
```
Comment (mykola-mokhnach) [inline lib/chromedriver.js]: I thought semver is able to coerce such version numbers properly %/

Comment (imurchie) [reply lib/chromedriver.js]: Semver only handles major, minor, and patch levels. Chromium (and now Chromedriver) uses four levels (not sure what their names are).

With the `semver` package `1.2.3.4` is the same as `1.2.3`.

Comment (mykola-mokhnach) [reply lib/chromedriver.js]: do we actually care about the last number there? This must be a build number. And I would not expect we have two different CD releases where only this number differs

Comment (imurchie) [reply lib/chromedriver.js]: Currently they have releases [72.0.3626.69](https://chromedriver.storage.googleapis.com/index.html?path=72.0.3626.69/) and [72.0.3626.7](https://chromedriver.storage.googleapis.com/index.html?path=72.0.3626.7/), as well as [73.0.3683.20](https://chromedriver.storage.googleapis.com/index.html?path=73.0.3683.20/) and [73.0.3683.68](https://chromedriver.storage.googleapis.com/index.html?path=73.0.3683.68/).

So I'm not particularly willing to take too many assumptions.

Comment (mykola-mokhnach) [reply lib/chromedriver.js]: fatality

File: lib/chromedriver.js [hunk 2]
```diff
@@ -189,25 +188,28 @@ class Chromedriver extends events.EventEmitter {
       if (!match) {
         return logError({message: 'Cannot parse the version string', stdout, stderr});
       }
+      let version = match[1];
       const versionObj = semver.coerce(match[1], true);
```
Comment (mykola-mokhnach) [inline lib/chromedriver.js]: coerce(version, true)

File: lib/chromedriver.js [hunk 3]
```diff
@@ -189,25 +188,28 @@ class Chromedriver extends events.EventEmitter {
...
-      if (!versionObj) {
-        return logError({message: 'Cannot coerce the version number', stdout, stderr});
+      if (versionObj) {
+        const tempVersion = `${versionObj.major}.${versionObj.minor}`;
```
Comment (mykola-mokhnach) [inline lib/chromedriver.js]: can we have more descriptive var name?

**Pull request merged**